.travis.yml

```diff
@@ -2,44 +2,44 @@ language: php
 
 php:
     - "5.6"
-    - "7.0"
-    - "7.1"
+    - "7.2"
+    - "7.3"
 
 services:
     - mysql
 
 env:
     - WP_VERSION=latest WP_MULTISITE=0 #Current stable release
 
-# Test WP 5.1 and 5.2 and on PHP 5.6, 7.0 and 7.1 (w and w/o multisite enabled)
+# Test WP 5.1 and 5.2 and on PHP 5.6, 7.2 and 7.3 (w and w/o multisite enabled)
 # Test WP master on PHP nightly and allow_failures (w/o multisite enabled)
 
 matrix:
   include:
     # current stable release w/ multisite
     - php: "5.6"
       env: WP_VERSION=5.1 WP_MULTISITE=0
-    - php: "7.0"
+    - php: "7.2"
       env: WP_VERSION=5.1 WP_MULTISITE=0
-    - php: "7.1"
+    - php: "7.3"
       env: WP_VERSION=5.1 WP_MULTISITE=0
     - php: "5.6"
       env: WP_VERSION=5.1 WP_MULTISITE=1
-    - php: "7.0"
+    - php: "7.2"
       env: WP_VERSION=5.1 WP_MULTISITE=1
-    - php: "7.1"
+    - php: "7.3"
       env: WP_VERSION=5.1 WP_MULTISITE=1
     - php: "5.6"
       env: WP_VERSION=5.2 WP_MULTISITE=0
-    - php: "7.0"
+    - php: "7.2"
       env: WP_VERSION=5.2 WP_MULTISITE=0
-    - php: "7.1"
+    - php: "7.3"
       env: WP_VERSION=5.2 WP_MULTISITE=0
     - php: "5.6"
       env: WP_VERSION=5.2 WP_MULTISITE=1
-    - php: "7.0"
+    - php: "7.2"
       env: WP_VERSION=5.2 WP_MULTISITE=1
-    - php: "7.1"
+    - php: "7.3"
       env: WP_VERSION=5.2 WP_MULTISITE=1
     - php: "nightly"
```

Comment:
I wonder if we should just remove php nightly? Not really that much benefit to testing it.

Reply:
Agree, if anything we should add PHP 7.4